charts/kured/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "1.6.1"
 description: A Helm chart for kured
 name: kured
-version: 2.3.1
+version: 2.4.0
```

Comment:
Updated this value at the suggestion of the helm lint CI, certainly I'll be guided by the maintainers as to what the correct version bump action should be here 🙇

Reply:
The helm chart is trailing the release of kured, so ideally you should have 2 PRs:
- one for the change in code of kured, and eventual manifests, which **won't** be exposed for end users until we officially release
- one PR for the change in the helm chart, which exposes the features through the helm chart.

As to the version itself, I would guess that 2.4.0 would be right, but the appVersion will most likely also be updated to 1.7.0.